### DISCUSSION TITLE
Responses in Body Json

### DISCUSSION THREAD

**Discussion by 5011-5011** [nextgenhealthcare/connect#5881] [Q&A] [answered]
Hello!
I would like to know if there is any way to:
Context:
1. I have a HTTP LISTENER channel, which receives messaging. 
2. At the request of the client who is the one who sends this messaging to my channel, he needs that the responses do not go in the headers, but he expects to receive them in a JSON in the body.

My question is: Is there a way to configure this from the "source" tab? 

![image](https://github.com/nextgenhealthcare/connect/assets/98954172/6460c527-1703-440d-ad66-6b4dacb4958d)

Today I have the answers saved in a channel map that I include in Headers.

**Comment by pacmano1** [comment, marked as answer]
Yes, that varaible you have at the top next to "Response:" would be the variable where you put the JSON.

i.e. "Resultado" would need JSON in it.

**Reply by 5011-5011** [reply]
I solved it a few minutes ago!
Thanks anyway

**Reply by pacmano1** [reply]
Also I not clear why you have all those response headers.  You are also duplicating stuff in the response header section.

**Reply by 5011-5011** [reply]
I know. This channel is for testing and included things to test others.
I haven't been with Mirth Connect that long, there are a lot of things I need to learn.
Thanks for the observation.
